experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -216,7 +216,23 @@ <h4>Audience</h4>
             </tr>
             <tr>
               <th>Recipe JSON</th>
-              <td colspan="3">{{ experiment.targeting }}</td>
+              <td colspan="3">
+                <div class="collapse" id="collapseRecipe">
+                  <code>
+                    <pre class="text-monospace" style="white-space: pre-wrap; word-wrap: break-word;">
+                      {{ experiment.recipe_json|linebreaks|linebreaksbr }}
+                    </pre>
+                  </code>
+                </div>
+                <button class="btn btn-outline-primary btn-sm"
+                        type="button"
+                        data-bs-toggle="collapse"
+                        data-bs-target="#collapseRecipe"
+                        aria-expanded="false"
+                        aria-controls="collapseRecipe">
+                  <i class="fa-solid fa-plus"></i> Show Recipe
```

Comment:
Can this be dynamic (Show vs Hide Recipe)? Or if not, maybe "Show/Hide Recipe" would be better?

Reply:
Yeah I couldn't find a way to easily make it dynamic.  Good idea to call it show/hide.